crates/apps_lib/src/cli.rs

```diff
@@ -6498,7 +6526,7 @@ pub mod args {
                     "The channel ID via which the token is received."
                 )))
                 .arg(REFUND.def().help(wrap!(
-                    "Generate the shielded transfer for refunding."
+                    "Generate the shielding transfer for refunding."
```

Comment:
This is used to generate both the refunding tx and the actual shielding transfer on the destination chain, correct? If that's the case maybe we can change the message here

Reply:
Thanks, Hermes used the argument. Now `build_ibc_transfer` generates the masp tx for the refunding.
Users doesn't have to generate it with this command anymore.
I'll remove this argument.